src/SPC/util/SPCConfigUtil.php

```diff
@@ -143,6 +143,10 @@ private function getIncludesString(array $libraries): string
             }
             $pc_cflags = implode(' ', $pc);
             if ($pc_cflags !== '' && ($pc_cflags = PkgConfigUtil::getCflags($pc_cflags)) !== '') {
+                $arr = explode(' ', $pc_cflags);
+                $arr = array_unique($arr);
```

Comment:
I'm not quite sure if there will have a CFLAGS that contains spaces like `-idirafter /path`. Simply unique to cflags seems not safe enough.

Reply:
I didn't find existing CFLAGS with spaces either, the only possibility is that the include paths contain spaces, but it will break autoconf first.